bench-vortex/src/bin/tpch_benchmark.rs

```diff
@@ -18,10 +18,13 @@ async fn main() {
     // The formats to run against (vs the baseline)
     let formats = [
         Format::Arrow,
-        Format::Parquet,
         Format::Vortex {
             disable_pushdown: false,
         },
+        Format::Vortex {
+            disable_pushdown: true,
+        },
+        // Format::Parquet,
```

Comment:
for some reason this is taking exceptionally long for me,  even on a beefy mbp. poking

Reply:
ok it's a bug, #526